.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug Report
+description: 
+title: "[Bug]"
+labels: bug
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for taking the time to report a bug!
+      In order to prevent drowning in unactionable bugs, maintainers will close any bugs that are not clearly defined. 
+      In particular, if you are having general issues with your particular build environment, 
+      you should instead start a conversation in [Discussions](https://github.com/visgl/deck.gl/discussions)
+      rather than opening issues with build logs etc. 
+      Please understand this repo is called deck.gl-community because it is maintained by the community,
+      (there are no dedicated maintainers) meaning that we mostly rely on people who report bugs to make PRs for the fixes.
+      We are however happy to help review and land your PRs, and publish new versions with the fixes.
+      Also check [existing issues](https://github.com/visgl/deck.gl-community/issues)to avoid filing duplicate bugs.
+      You may find answers faster by searching in [the documentation](https://deck.gl-community/search).
+- type: checkboxes
```

Comment:
Should we add a `I plan to submit a PR` checkbox to make clear which issue the requester plans to work on?

Reply:
Yes... I suppose it is good to know if the reporting is planning to try to fix it.